api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "edge": {
             "version_added": null
           },
           "edge_mobile": {
             "version_added": null
           },
-          "firefox": {
-            "version_added": null
-          },
+          "firefox": [
+            {
+              "version_added": "57"
+            },
+            {
+              "version_added": "51",
+              "version_removed": "57",
+              "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.storageManager.enabled",
+                  "value_to_set": "true"
```

Comment:
MDN stated that, from version 55, the preference `browser.storageManager.enabled` was also required, but I found no evidence for this (neither in the repository nor in the bugs).